webroot/css/d3.flameGraph.css

```diff
@@ -32,47 +31,14 @@
 
 .d3-flame-graph .title {
   font-size: 20px;
+  font-family: Verdana;
 }
 
 .d3-flame-graph .label {
   background: transparent;
 }
 
-.d3-flame-graph-tip {
-  line-height: 1;
-  padding: 12px;
-  background: #fff;
-  border-radius: 6px;
-  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.2);
-  border: 1px solid rgba(0, 0, 0, 0.2);
-  color: #000;
-  pointer-events: none;
-}
-
-/* Creates a small triangle extender for the tooltip */
-.d3-flame-graph-tip:before,
-.d3-flame-graph-tip:after {
-    pointer-events: none;
-    content: '';
-    position: absolute;
-    display: block;
-    width: 0;
-    height: 0;
-    border-width: 11px;
-    border-color: transparent;
-    border-style: solid;
-    left: 50%;
-    z-index: 2;
-}
-.d3-flame-graph-tip:after {
-    border-bottom-color: #fff;
-    margin-left: -11px;
-    top: -22px;
-    z-index: 2;
-}
-.d3-flame-graph-tip:before {
-    border-width: 12px;
-    border-bottom-color: #999;
-    margin-left: -12px;
-    top: -24px;
-}
+/**
+ * PATCH(xhgui): Remove all .d3-flame-graph-tip styles
+ * We have our own tip styles in xhgui.css instead.
```

Comment:
Do we need this comment long term?

Reply:
I think it'd make sense to keep yeah.

I'm working on another pull request that updates d3 to version 4, and updates d3-tip and d3-flameGraph. As part of such file-replacing commits, seeing the two `PATCH` comments in the removal diff is a reminder that the file was modified and needs to be re-modified the same way.

Ideally we wouldn't have these patches (undocumented fork, modification of lib files) as it wouldn't be possible in the first place if this were installed via npm or bower. But given we have the local patch for now, rather than the file just missing it and accidentally adding it back when updating the file, we need to document it.

Alternatively, I could move the d3-flameGraph to a separate directory with a `PATCHES` file. If you prefer that.